PKGBUILD

```diff
@@ -0,0 +1,22 @@
+# Maintainer: Your Name <youremail@domain.com>
+pkgname="particle-life"
+pkgrel=1
+pkgver=v1.1.3 # todo: make automatic
```

Comment:
According to [this](url), you can automate this by defining a `pkgver()` function in the PKGBUILD
Example from https://wiki.archlinux.org/title/VCS_package_guidelines#The_pkgver()_function:
```sh
pkgver() {
  cd "$pkgname"
  git describe --long | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
}
```